cpp/src/parquet/column_reader.cc

```diff
@@ -1268,7 +1269,7 @@ class TypedRecordReader : public ColumnReaderImplBase<DType>,
       records_read += ReadRecordData(num_records);
     }
 
-    int64_t level_batch_size = std::max(kMinLevelBatchSize, num_records);
+    int64_t level_batch_size = std::max<int>(kMinLevelBatchSize, num_records);
```

Comment:
same comment on type-template param for max

Reply:
done